knowledge-base/form-focus-formitem.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Set Programmatic Focus to a FormItem
+description: How to set programmatic focus to an item in the Telerik Form.
+type: how-to
+page_title: How to set Programmatic Focus to a FormItem
+slug: form-kb-focus-formitem
+position: 
+tags: telerik, blazor, ref
+ticketid: 1592333
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>Telerik UI for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+## Description
+
+How to set programmatic focus to a `<FormItem>`?
```

Comment:
Since we don't focus the `FormItem` itself, but an input component inside it, we can enhance the article description, so that it contains more keywords. For example, add another line, which says "How to focus a specific input component in the form by default" or any variation of that.